### ISSUE TITLE
Error on data download

### ISSUE THREAD

**Issue by kkoch** [glos/GLBuoys#35]
I was testing some data downloads for a user query on GLBuoys and get either a general "500 Server Error" or the message below.

Attempted a back range of dates that I confirmed on tds we should cover and also on last 30 days on one that was recently pulled out of the water:

Internal Server Error
The server encountered an internal error or misconfiguration and was unable to complete your request.
Please contact the server administrator at root@localhost to inform them of the time this error occurred, and the actions you performed just before this error.
More information about this error may be available in the server error log.

**Comment by gcutrell** [comment]
@kkoch If you could provide the permalink that would be helpful.

**Comment by gcutrell** [comment]
@kkoch Thanks. So it isn't able to handle selected dates that are beyond the end of record.

**Comment by kkoch** [comment]
Nope.  It also gives me the same error when I put in a range that I know (by checking tds) that we have.

https://glbuoys.glos.us/tools/download_data?ftype=xls&units=eng&data_type=buoy&locs=45013&params=Wind_Speed&tperiod=custom&date_start=2018-11-01&date_end=2018-11-05&avg_ivld=none

Maybe the server needs kicked?
